workflow/schemas/config.schema.yaml

```diff
@@ -2,20 +2,84 @@ $schema: "http://json-schema.org/draft-04/schema#"
 description: snakemake configuration file
 type: object
 properties:
-  alignment_software:
+  resources:
     type: string
+    description: path to resources.yaml file
+
   samples:
     type: string
-  trimmer_software:
-    type: string
-    description: Set input files for alignment software, if "None" is set the files in units.tsv will be used.
-    pattern: "^(fastp_pe|None)$"
+    description: path to samples.tsv file
+
   units:
     type: string
+    description: path to units.tsv file
+
   default_container:
     type: string
+    description: name or path to a default docker/singularity container
+
+  alignment_software:
+    type: string
+    description: alignment software that will be used. Default is bwa
+    pattern: "^(gpu|bwa|None)$"
```

Comment:
What is gpu doing?

Reply:
I also think `None` does not make sense as this module is for alignment and we need to yield some kind of `.bam`-file, right? But maybe we can discuss the reasoning behind this.

Reply:
we should then maybe remove the option of settings this, since we right now only have one alignment software?

Reply:
Agreed.